.scribe/intro.md

```diff
@@ -3,7 +3,7 @@
 The documentation
 
 <aside>
-    <strong>Base URL</strong>: <code>https://staging-api.learnhub.mk</code>
```

Comment:
You have to add the APP_STAGING_URL .env variable and assign it to this URL in order for the docs to generate with the correct URL and the "try it out" feature to work.

Reply:
I added APP_STAGING_URL but it still fails it requires 

# social accounts
GITHUB_CLIENT_ID=
GITHUB_CLIENT_SECRET=
GITHUB_REDIRECT_URI=

GOOGLE_CLIENT_ID=
GOOGLE_CLIENT_SECRET=
GOOGLE_REDIRECT_URI=

LINKEDIN_CLIENT_ID=
LINKEDIN_CLIENT_SECRET=
LINKEDIN_REDIRECT_URI=

Reply:
Fails on generation? Add some random values and see if it will fix it.

Reply:
It doesn't work with dummy text. I will need to create accounts for socillite account and then create scribe docs for this section. I added update scribe docs with new values for other routes and header type. There were some bugs I fixed them for scribe:generate